frontend/src/AddSeries/SeriesMonitorNewItemsOptionsPopoverContent.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import DescriptionList from 'Components/DescriptionList/DescriptionList';
+import DescriptionListItem from 'Components/DescriptionList/DescriptionListItem';
+import translate from 'Utilities/String/translate';
+
+function SeriesMonitorNewItemsOptionsPopoverContent() {
+  return (
+    <DescriptionList>
+      <DescriptionListItem
+        title={translate('MonitorAllEpisodes')}
+        data="Monitor all new episodes"
```

Comment:
I think this is a bit misleading because episodes should follow the season's monitored state and this should be more about monitoring new seasons.

Having this also control episodes is going to make the interaction with `MonitoringOptions` confusing as well.

Reply:
Yea, makes sense to change to seasons for Sonarr, all the others are just date based releases with no grouping